armi/bookkeeping/db/database3.py

```diff
@@ -275,7 +275,7 @@ def interactDistributeState(self):
         """
         if armi.MPI_RANK > 0:
             # DB may not exist if distribute state is called early.
-            if os.path.exists(self._dbPath):
+            if self._dbPath is not None and os.path.exists(self._dbPath):
```

Comment:
Almost feels like we could have a Database3.dbPathIsValid() class method or something in case this chain gets longer. But it probably won't get much longer...

Reply:
Perhaps. Fact of the matter is `_dbPath` is an `Optional[Pathlike]`, so it's not safe to use it as a path-like without checking for None. I'm not sure why mypy isn't catching this one, to be honest....